web/packages/teleterm/src/ui/DocumentTerminal/useDocumentTerminal.ts

```diff
@@ -389,6 +391,40 @@ function createCmd(
     };
   }
 
+  if (doc.kind === 'doc.gateway_kube') {
+    const gateway = clustersService.findGatewayByConnectionParams(
+      doc.targetUri,
+      ''
+    );
+    if (!gateway) {
+      throw new Error(`No gateway found for ${doc.targetUri}`);
+    }
+
+    const env = tshdGateway.getCliCommandEnv(gateway.gatewayCliCommand);
+
+    if ('KUBECONFIG' in env === false) {
+      // This shouldn't happen as 'KUBECONFIG' is the sole purpose of the CLI
+      // command for a kube gateway.
+      throw new Error(
+        `No KUBECONFIG provided for gateway ${gateway.targetUri}`
+      );
+    }
+    const helpMsg =
+      "Started local proxy for Kubernetes cluster '" +
+      gateway.targetName +
+      "' and environment variable 'KUBECONFIG' is updated to the kubeconfig provided by the local proxy.\r\n\r\n" +
+      "Try 'kubectl version' to test the connection.\r\n\r\n";
```

Comment:
```suggestion
    const helpMsg =
      `Started a local proxy for Kubernetes cluster "${gateway.targetName}".\r\n\r\n` +
      'The KUBECONFIG env var can be used with third-party tools as long as the proxy is running.\r\n' +
      'Close the proxy from Connections in the top left corner or by closing Teleport Connect.\r\n\r\n' +
      "Try 'kubectl version' to test the connection.\r\n\r\n";
```

I feel like for a regular user, this will do a better job at explaining how it all works.